test/integration-tests/SwiftSnippet.test.ts

```diff
@@ -0,0 +1,91 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the VS Code Swift open source project
+//
+// Copyright (c) 2024 the VS Code Swift project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of VS Code Swift project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import * as vscode from "vscode";
+import { folderContextPromise } from "./extension.test";
+import { testAssetPath, testAssetUri } from "../fixtures";
+import { waitForNoRunningTasks } from "../utilities/tasks";
+import { expect } from "chai";
+import {
+    continueSession,
+    waitForDebugAdapterCommand,
+    waitForDebugAdapterExit,
+    waitUntilDebugSessionTerminates,
+} from "../utilities/debug";
+import { Version } from "../../src/utilities/version";
+
+suite("SwiftSnippet Test Suite", () => {
+    const uri = testAssetUri("defaultPackage/Snippets/hello.swift");
+    const breakpoints = [
+        new vscode.SourceBreakpoint(new vscode.Location(uri, new vscode.Position(2, 0))),
+    ];
+
+    suiteSetup(async function () {
+        const folder = await folderContextPromise("defaultPackage");
+        if (folder.workspaceContext.toolchain.swiftVersion.isLessThan(new Version(6, 0, 0))) {
```

Comment:
IIRC the source specified 5.7 for snippet feature, something in the CI hindering us I guess?